Dockerfile

```diff
@@ -23,6 +23,7 @@ RUN set -ex; \
     curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs >/tmp/rustup.sh; \
     sh /tmp/rustup.sh -y --no-modify-path --default-toolchain "$TOOLCHAIN"; \
     rm /tmp/*.sh
+RUN rustup install nightly
```

Comment:
AFAIK we don't need it because we `  RUSTC_BOOTSTRAP: 1`

Reply:
If we do install nightly inside the container, we need to pin its version.